### PR TITLE
Switch to zola 0.19.2 and fix generate_feeds config.

### LINKED ISSUE #62: Site fails to build with latest version of Zola
Trying to build with Zola 0.19.2 gives this error:

```
Building site...
Error: Failed to serve the site
Error: TOML parse error at line 13, column 1
   |
13 | generate_feed = true
   | ^^^^^^^^^^^^^
unknown field `generate_feed`, expected one of `base_url`, `theme`, `title`, `description`, `default_language`, `languages`, `translations`, `generate_feeds`, `feed_limit`, `feed_filenames`, `hard_link_static`, `taxonomies`, `author`, `compile_sass`, `minify_html`, `build_search_index`, `ignored_content`, `ignored_static`, `mode`, `output_dir`, `preserve_dotfiles_in_output`, `link_checker`, `slugify`, `search`, `markdown`, `extra`, `generate_sitemap`, `generate_robots_txt`
```

From a quick check, CI currently uses version 0.17.2. This should at least be documented. Ideally we should port to the latest version.

### PR DESCRIPTION
**Pull request by PoignardAzur**
Fixes #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,6 @@ available on the website.
 
 # Development
```
Comment (DJMcNab) [inline README.md:5]:
```suggestion

**Note: The site is currently built with Zola version 0.19.2.**

```
Although given you ignored this same suggestion in #63, I'm presuming there's some reason this is bad.

I would however like to be told that

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ jobs:
       - name: checkout
         uses: actions/checkout@v3.0.0
       - name: build_and_deploy
-        uses: shalzz/zola-deploy-action@v0.17.2
+        uses: shalzz/zola-deploy-action@v0.19.2
```
Comment (DJMcNab) [inline .github/workflows/main.yml:16]:
```suggestion
        # Also update in README.md and ./check.yml
        uses: shalzz/zola-deploy-action@v0.19.2
```

File: .github/workflows/check.yml [hunk 3]
```diff
@@ -13,6 +13,6 @@ jobs:
       - name: checkout
         uses: actions/checkout@v3.0.0
       - name: build_and_deploy
-        uses: shalzz/zola-deploy-action@v0.17.2
+        uses: shalzz/zola-deploy-action@v0.19.2
```
Comment (DJMcNab) [inline .github/workflows/check.yml:16]:
```suggestion
        # Also update in README.md and ./main.yml
        uses: shalzz/zola-deploy-action@v0.19.2
```

**Review by DJMcNab — COMMENTED** [review]
Thanks! Good to get this updated, and it's nice to see that the changes are relatively small

**Comment by PoignardAzur** [comment]
> Although given you ignored this same suggestion in https://github.com/linebender/linebender.github.io/pull/63, I'm presuming there's some reason this is bad.

Nah, I was just feeling too tired to consider it.

**Pull request merged**